QsLogDestFile.cpp

```diff
@@ -24,7 +24,7 @@
 // OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "QsLogDestFile.h"
-#include <QTextCodec>
+#include <QtCore5Compat/QTextCodec>
```

Comment:
is this still needed, given that you're not using QTextCodec any more?  would be good if you could avoid linking against core5compat at all